frontends/web/src/locales/en/app.json

```diff
@@ -588,11 +589,16 @@
   },
   "seedRestore": {
     "info": {
-      "button": "Continue",
-      "description": "To continue please insert the micro SD card in to the BitBox.",
-      "title": "How to restore a wallet from a backup"
+      "title": "Restore a wallet",
+      "description_1": "Insert the micro SD card in to the BitBox and click “Continue”",
```

Comment:
Please don't use `_` in keys, unless it is used as dynamic context. e.g. `t('my.key', { context: 'foo' })` resolves to `my.key_foo`.

Reply:
tried to fix, but some are coming from the backend. i.e. `unlock.error.e109_normal`

Reply:
Yes, that is a valid use of context, see [here](https://github.com/digitalbitbox/bitbox-wallet-app/blob/c843d712207594a567f785ba083f2980fc5a6522/frontends/web/src/routes/device/unlock.jsx#L127). Thanks for doing the rest :)